arcgis-ios-sdk-samples/Layers/List KML contents/ListKMLContentsViewController.swift

```diff
@@ -0,0 +1,113 @@
+// Copyright 2018 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class ListKMLContentsViewController: UITableViewController {
+    
+    private var kmlDataset: AGSKMLDataset?
+    
+    /// The full node tree flattened to an array of tuples that include each node's level.
+    /// Used as the model for the table view.
+    private var flattenedNodes: [(node: AGSKMLNode, level: Int)] = []
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        guard self.kmlDataset == nil else {
```

Comment:
I'm a bit confused by this. Is `kmlDataset` ever set elsewhere? If not, this check isn't needed. `viewDidLoad()` is only ever called once per instance, so the expression would never evaluate to `false`.

Reply:
I don't even remember writing that line haha